docs/GoogleDrive-batchsink.md

```diff
@@ -0,0 +1,26 @@
+# Google Drive Batch Sink
+
+
+Description
+-----------
+Sink plugin saves files from the pipeline to Google Drive directory via Google Drive API.
+The minimal input schema contains only byte array with field name configured by **schemaBodyFieldName** property. 
```

Comment:
schemaBodyFieldName
and
schemaNameFieldName

And core properties of the plugin. Which represent how it works I think they should be at the top of widget and doc not at the bottom.

Also move this description to properties description.

Reply:
Also add information about required type of the schema fields.

Reply:
done